config/locales/server.en.yml

```diff
@@ -754,6 +755,7 @@ en:
               reply_by_email_disabled: "Reply by email has been disabled."
               send_to_email_disabled: "Sorry, you cannot send personal messages to email."
               target_user_not_found: "One of the users you are sending this message to could not be found."
+              too_large_group: "The group: %{group_name}, has too many users to receive a message."
```

Comment:
This seems to be the actual error shown to the users but we don't have the max count in it, we should add it I think.

Reply:
But maybe it's not possible in topic creator? IDK...

Reply:
It is possible since I added my changes, I thought it was maybe too much details for end user.